docs/source/training_pipeline/index.rst

```diff
@@ -0,0 +1,15 @@
+:orphan:
+
+.. _training_pipeline:
+
+***********************
+Training your own model
```

Comment:
Wrong stylization compared with the other Tutorials titles, see image attached:
![image](https://user-images.githubusercontent.com/13075447/229012912-7af0f1c3-2f13-432d-b598-9760b04baba0.png)

Reply:
Will update the words as capitalized and remove the expand button.